resources/assetstore/charts/asset-upload-service/README.md

```diff
@@ -0,0 +1,9 @@
+# Asset Upload Service
+
+## Overview
+
+The Asset Upload Service is an HTTP server that exposes the file upload functionality for Minio. It contains a simple HTTP endpoint which accepts multipart/form-data forms. It can upload files to the private and public system buckets.
+
+## Details
+
+To learn more about the Asset Upload Service, see the [Asset Upload Service documentation](https://kyma-project.io/docs/components/asset-store#details-asset-upload-service) on [kyma-project.io](https://kyma-project.io/).
```

Comment:
```suggestion
To learn more about the Asset Upload Service, see the [Asset Upload Service documentation](https://kyma-project.io/docs/components/asset-store#details-asset-upload-service).
```

Reply:
I am thinking if there is a need for the **Details** section at all if we only link in it to the official docs?